lib/matplotlib/legend_handler.py

```diff
@@ -255,6 +256,62 @@ def create_artists(self, legend, orig_handle,
         return [legline, legline_marker]
 
 
+class HandlerLine2D(HandlerNpoints):
+    """
+    Handler for `.Line2D` instances.
+
+    A class similar to the original handler for `.Line2D` instances but
+    that uses a monolithic artist rather than using one artist for the
+    line and another one for the marker(s).  NB: the former handler, in
```

Comment:
Perhaps use one of .. note:: or .. versionadded:: or .. seealso:: (http://www.sphinx-doc.org/en/stable/markup/para.html#paragraph-level-markup).  The latter could also be a See also numpydoc section.